kpops/utils/dict_differ.py

```diff
@@ -53,9 +57,9 @@ def from_dicts(
         d1: dict, d2: dict, ignore: set[str] | None = None
     ) -> Iterator[Diff]:
         for diff_type, keys, changes in diff(d1, d2, ignore=ignore):
-            if not isinstance(changes, list):
-                changes = [("", changes)]
-            for key, change in changes:
+            if not isinstance(changes_tmp := changes, list):
+                changes_tmp = [("", changes)]
+            for key, change in changes_tmp:
```

Comment:
could you explain this change?

Reply:
[PLW2901](https://docs.astral.sh/ruff/rules/redefined-loop-name/) complained. I was not sure whether to suppress it, but ended up following the advice.